### DISCUSSION TITLE
Acessos para nova organização da Python Sudeste (2025)

### DISCUSSION THREAD

**Discussion by vbuxbaum** [apyb/comunidade#140] [Geral]
Olá pessoal! Tudo bem por aí?

A Python Sudeste 2025 será realizada em Belo Horizonte - MG, mas a nova equipe ainda não possui os acessos de email, rede social, etc. para dar continuidade a partir do que foi realizado para edições anteriores.

Podem me ajudar com isso? ❤️ Eu (Vitor Buxbaum Orlandi) serei o big-kahuna, mas temos @jeanmask @debthomaz e outras pessoas envolvidas.

Desde já, muito obrigado!

cc: @anadulce

**Comment by cecivieira** [comment]
Nós da Apyb não temos esses acessos.

**Comment by tiidadavena** [comment]
> Olá pessoal! Tudo bem por aí?
> 
> A Python Sudeste 2025 será realizada em Belo Horizonte - MG, mas a nova equipe ainda não possui os acessos de email, rede social, etc. para dar continuidade a partir do que foi realizado para edições anteriores.
> 
> Podem me ajudar com isso? ❤️ Eu (Vitor Buxbaum Orlandi) serei o big-kahuna, mas temos @jeanmask @debthomaz e outras pessoas envolvidas.
> 
> Desde já, muito obrigado!
> 
> cc: @anadulce

Te add e Jean no grupo do Telegram da Python Sudeste. Quem tem os acessos são os Big Kahunas deste ano: @anadulce e @renan-asantos.

**Comment by rougeth** [comment]
@vbuxbaum não deixe de ler o guia de como usar a APyB como infraestrutura para o seu evento. Vai ficar mais claro como a APyB pode te ajudar, as condições e regras, movimentação financeira, etc:

https://apyb.python.org.br/associados/guias/como-organizar-eventos/

**Comment by vbuxbaum** [comment]
Obrigado pessoal!

@tiidadavena O grupo do telegram que você se refere é o https://web.telegram.org/k/#@pythonsudeste ? Se for outro grupo talvez tenha adicionado a pessoa errada... Meu usuário lá é `buxbaum`

@rougeth Obrigado! Já fiz a leitura e tive o papo inicial com a diretoria da APYB ❤️

**Reply by naanadr** [reply]
@vbuxbaum te lembrando que caso a PySE não tenha ainda um chaveiro de senha, estamos recomendo utilizar o 1password. [Aqui](https://github.com/apyb/comunidade/discussions/130) nessa outra thread tem um pouco de como funcionou a criação do chaveiro para a PyNE. 

Qualquer dúvida sobre esse assunto, só perguntar.
